src/pkg/clouds/azure/aca/cert.go

```
@@ -181,6 +185,21 @@ func waitForBYODdns(ctx context.Context, hostname, expectedCname, expectedTxt st
 }
 
+// environmentStaticIP fetches the Container Apps environment's inbound static IP
+// for display in the apex A-record instructions. This is a JIT lookup done only
+// once the DNS-not-ready prompt is about to be shown, so the common case (DNS
+// already configured) never pays for it. A lookup failure falls back to a
+// placeholder rather than failing cert issuance — the value is purely
+// informational; dns.CheckDomainDNSReady validates the actual DNS state itself.
+func environmentStaticIP(ctx context.Context, envsClient *armappcontainers.ManagedEnvironmentsClient, resourceGroup, envName string) string {
+	env, err := envsClient.Get(ctx, resourceGroup, envName, nil)
+	if err != nil || env.Properties == nil || env.Properties.StaticIP == nil {
+		term.Debugf("Could not fetch static IP for environment %s: %v", envName, err)
```

golangci-lint (2.12.2)

[error] Error return value of `term.Debugf` is not checked

(errcheck)